coreosBuilder.py

```diff
@@ -70,15 +71,19 @@ class CoreosBuilder():
     def __init__(self, working_dir: str):
         self._workdir = working_dir
 
-    def build(self, dst: str) -> None:
+    def build(self, dst: str, is_bf: Optional[bool] = True) -> None:
```

Comment:
I prefer not to have a specific image for is_bf for the sake of simplicity. You can add your libvirtd stuff to the single image we build.